src/Loaders/LoadingSpinner.stories.mdx

```diff
@@ -21,19 +21,19 @@ import { colors } from "../colors";
 
 <Preview>
   <Story name="size large">
-    <LoadingSpinner size="large" />
+    <LoadingSpinner size="large" syncAnimationRestarts={false} />
```

Comment:
Q: Should all of these use `isChromatic()` instead of `false`?

Reply:
🤷‍♂Im not sure it matters either way, the spinner will just have a different offset when running locally